specification/securityinsights/data-plane/Microsoft.SecurityInsights/preview/2024-02-01-preview/ThreatIntelligence.json

```
@@ -0,0 +1,189 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Sentinel Upload API",
+    "description": "Specification for Microsoft Sentinel Upload API.",
+    "version": "2024-02-01-preview"
+  },
+  "host": "api.ti.sentinel.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/workspaces/{workspaceId}/threatintelligencestixobjects:upload": {
```

Please use kebab- or camel-case for multi-word static path segments.
```suggestion
    "/workspaces/{workspaceId}/threat-intelligence-stix-objects:upload": {
```